### PR TITLE
Task/recover password

### PR DESCRIPTION
**Pull request by JuanGro**
#### What's this PR do?
To implement recover password method.

#### Where should the reviewer start?
With the recover password form.

#### How should this be manually tested?
Recovering your password.

#### Any background context you want to provide?
No.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: src/app/login/recover-password-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+// Environment
+import { environment } from '../../environments/environment';
+
+@Component({
+    selector: 'recover-password-form',
+    templateUrl: 'recover-password-form.component.html'
+})
+export class RecoverPasswordFormComponent implements OnInit {
+    // Login object for the ngForm handle.
+    public recoverPassword: Login;
```
Comment (EduardoVaca) [inline src/app/login/recover-password-form.component.ts]: Choose a better variable name. It's confusing that a Login object is calles recoverPassword.

File: src/environments/environment.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@
 export const environment = {
   production: false,
   'API_URL': 'http://127.0.0.1:8000/api/',
+  'HOME_URL': 'designer/owned-designs-list',
```
Comment (EduardoVaca) [inline src/environments/environment.ts]: This URL shouldn't be here, these are just Django's API urls. HOME_URL is from angular's routing, so please remove it from here.

File: src/app/login/recover-password-form.component.ts (hunk 1 above)
Comment (JuanGro) [inline src/app/login/recover-password-form.component.ts]: Done

File: src/environments/environment.ts (hunk 2 above)
Comment (JuanGro) [inline src/environments/environment.ts]: Done

**Pull request merged**